runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -811,7 +975,9 @@ Java_java_lang_invoke_MethodHandleNatives_resolve(
 	const J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
 	jobject result = NULL;
 	J9UTF8 *name = NULL;
+	char nameBuffer[256] = {0};
 	J9UTF8 *signature = NULL;
+	char signatureBuffer[256] = {0};
```

Comment:
Just the first element of these arrays should be initialized; doing more is just a waste of time.